decidim-module-census_connector/app/controllers/concerns/decidim/census_connector/census_context.rb

```diff
@@ -9,7 +9,11 @@ module CensusContext
       extend ActiveSupport::Concern
 
       included do
-        helper_method :local_scope, :local_scope_ranges, :has_person?, :person, :person_participatory_spaces
+        helper_method :document_scopes, :local_scope, :local_scope_ranges, :has_person?, :person, :person_participatory_spaces
+
+        def document_scopes
+          current_organization.scopes.where(parent_id: nil).order(name: :asc)
```

Comment:
just a question: why `current_organization.scopes.where(parent_id: nil).order(name: :asc)` and not `current_organization.scopes.top_level.order(name: :asc)`?

Reply:
Because I'm stupid :rofl: